frontend/src/metabase/search/components/SearchResult.jsx

```diff
@@ -1,62 +1,95 @@
 import React from "react";
 import { Box, Flex } from "grid-styled";
 import styled from "styled-components";
+import { jt } from "ttag";
 
 import * as Urls from "metabase/lib/urls";
-import { color } from "metabase/lib/colors";
+import { color, lighten } from "metabase/lib/colors";
 
 import Icon from "metabase/components/Icon";
 import Link from "metabase/components/Link";
-import Tooltip from "metabase/components/Tooltip";
+import Text from "metabase/components/type/Text";
+
+import Database from "metabase/entities/databases";
+import Table from "metabase/entities/tables";
+
+function colorForType(props) {
+  if (props.item.collection_position) {
+    return color("warning");
+  }
+  switch (props.type) {
+    case "collection":
```

Comment:
Could probably make this a simple if rather than a `switch` since for now we only check for collection to change the color.